src/tqec/templates/constructions/qubit.py

```diff
@@ -26,6 +26,10 @@ class ComposedTemplateWithSides(ComposedTemplate):
     def get_plaquette_indices_on_sides(self, sides: list[TemplateSide]) -> list[int]:
         pass
 
+    @abstractmethod
+    def get_corresponding_mid_for_side(self, side_index: int) -> int:
+        pass
+
```

Comment:
I needed a long time to actually understand what this did. Documentation would be welcomed.

Reply:
I'm not sure if this is the correct way to do it, I think getting the plaquettes from the pipe is the better idea.